system/Format/XMLFormatter.php

```diff
@@ -66,7 +66,8 @@ public function format($data)
 			// @codeCoverageIgnoreEnd
 		}
 
-		$output = new \SimpleXMLElement('<?xml version="1.0"?><response></response>');
+		$options = $config->formatterOptions['application/xml'] ?? 0;
+		$output = new \SimpleXMLElement('<?xml version="1.0"?><response></response>',$options);
```

Comment:
```suggestion
		$output = new \SimpleXMLElement('<?xml version="1.0"?><response></response>', $options);
```
Sorry, I missed this one.